spec/factories/loans.rb

```diff
@@ -55,11 +55,12 @@
 
 
     trait :active do
-      status_option_id Loan::STATUS_OPTIONS.value_for('Active')
+      # note: can't seem to count on the seed.db data having been loaded when these are executed
+      status_option_id 1 #Loan.status_option_set.value_for_label('Active')
```

Comment:
I agree it's not ideal for the specs to rely on seeds. Here's a partial explanation of why:
http://mikepackdev.com/blog_posts/13-seeding-data-to-your-rails-tests-with-factory-girl
It's basically the same argument as factories vs fixtures. If you haven't seen that:
https://semaphoreci.com/blog/2014/01/14/rails-testing-antipatterns-fixtures-and-factories.html
Trust me, I feel your pain. My least favorite part of TDD/BDD is writing code to generate fake data. But it's the better way to do it. And it does make it easier and nicer if you leverage FactoryGirl. The code sample Mel gave you looks really good to me. As I said above, however, I'm leaning toward removing the dependency on these `trait`s altogether, which would make it moot. :cow2: